it/v1/endpoints/AuthISpec.scala

```diff
@@ -0,0 +1,130 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package v1.endpoints
+
+import com.github.tomakehurst.wiremock.stubbing.StubMapping
+import play.api.http.HeaderNames.ACCEPT
+import play.api.http.Status
+import play.api.http.Status.NO_CONTENT
+import play.api.libs.ws.{WSRequest, WSResponse}
+import play.api.test.Helpers.AUTHORIZATION
+import shared.stubs.{AuditStub, AuthStub, DownstreamStub, MtdIdLookupStub}
+import support.IntegrationBaseSpec
+
+class AuthISpec extends IntegrationBaseSpec {
+
+  private trait Test {
+    val nino    = "AA123456A"
+    val taxYear = "2023-24"
+
+    def setupStubs(): StubMapping
+    def uri: String = s"/$nino/$taxYear"
+
+    def ifsUri: String = s"/income-tax/income/foreign/23-24/$nino"
+
+    def request(): WSRequest = {
+      setupStubs()
+      buildRequest(uri)
+        .withHttpHeaders(
+          (ACCEPT, "application/vnd.hmrc.1.0+json"),
+          (AUTHORIZATION, "Bearer 123") // some bearer token
+        )
+    }
+
+  }
+
+  "Calling the sample endpoint" when {
+
+    "MTD ID lookup fails with a 500" should {
+
+      "return 500" in new Test {
+        override val nino: String = "AA123456A"
+
+        override def setupStubs(): StubMapping = {
+          AuditStub.audit()
+          MtdIdLookupStub.error(nino, Status.INTERNAL_SERVER_ERROR)
+        }
+
+        val response: WSResponse = await(request().delete())
+        response.status shouldBe Status.INTERNAL_SERVER_ERROR
+      }
+    }
+
+    "an MTD ID is successfully retrieve from the NINO and the user is authorised" should {
```

Comment:
I think the name of the test is a bit misleading. It says "an MTD ID is successfully retrieve.. " but the next sub-test says "MTD ID lookup fails with a 403" . I think it should follow the naming as in the shared code here:
https://github.com/hmrc/self-assessment-bsas-api/blob/9f0ffa9931943f171200a4cab4c346380b7baac8/it/v3/endpoints/AuthISpec.scala#L75